apidef/api_definitions.go

```diff
@@ -135,6 +135,12 @@ type MethodTransformMeta struct {
 	ToMethod string `bson:"to_method" json:"to_method"`
 }
 
+type ValidatePathMeta struct {
+	Path         string `bson:"path" json:"path"`
+	Method       string `bson:"method" json:"method"`
+	ValidateWith string `bson:"validate_with" json:"validate_with"`
```

Comment:
Why not allow this to be a file path too, like we do with others?

If you're only going to allow a base64 string, you can use `[]byte` as a type and `encoding/json` already uses base64 implicitly.